package.json

```diff
@@ -1,6 +1,9 @@
 {
   "name": "slackin",
   "version": "0.3.1",
+  "scripts": {
+    "postinstall": "make"
```

Comment:
The problem with this is that the NPM version we publish already contains the precompiled source.